app/controllers/products_controller.rb

```diff
@@ -112,6 +112,7 @@ def get_related_product_sorted(product)
       params[:sort_by] = "oldest"
       @products = product.find_related_on_tags.sort_by(&:created_at)
     end
+    raise
```

Comment:
Please remove this.
![image](https://user-images.githubusercontent.com/78134040/131431192-2d3ec064-5faa-4762-9a23-4f05dd104885.png)